flutter/LICENSE

```diff
@@ -0,0 +1,22 @@
+Fluent Mobile Icons
```

Comment:
Is this file necessary? We don't have it in the iOS/Android libraries.

Reply:
Not needed, removed in 0e891dd